app/config/test.json

```diff
@@ -8,16 +7,7 @@
   "PGPORT": 5432,
   "PGDATABASE": "ccbc",
   "PGSCHEMA": "ccbc_public",
-  "CLIENT_SECRET": "",
-  "HOST": "",
-  "HAPPO_API_KEY": "",
-  "HAPPO_API_SECRET": "",
-  "NEXT_PUBLIC_GROWTHBOOK_API_KEY": "",
-  "AWS_S3_BUCKET": "",
-  "AWS_S3_REGION": "us-west-2",
-  "AWS_S3_KEY": "",
-  "AWS_S3_SECRET_KEY": "",
-  "AWS_ROLE_ARN": "",
+  "OPENSHIFT_APP_NAMESPACE": "ff61fb-test",
   "SITEMINDER_LOGOUT_URL": "https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi",
   "ENABLE_MOCK_TIME" : true,
   "ENABLE_MOCK_COOKIES" : true
```

Comment:
same comment as for `dev.json`, not sure we need anything else than those three.

Reply:
Done